src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -505,6 +566,66 @@ describe('ThreadView', () => {
         .not
         .toBeInTheDocument();
     });
+
+    it('pressing load more button will load next page of comments', async () => {
+      await waitFor(() => renderComponent(discussionPostId));
+      await mockAxiosReturnPagedComments(discussionPostId, false, 2);
+
+      const loadMoreButton = await findLoadMoreCommentsButton();
+      fireEvent.click(loadMoreButton);
```

Comment:
wrap this in act. The act function ensures that all updates triggered by the event are processed and any resulting asynchronous behavior is properly handled. so it is recommended to wrap fireEvent.click with the act.
await act(async () => {
  fireEvent.click(loadMoreButton);
});